server/index.js

```diff
@@ -54,6 +54,23 @@ io.on('connection', (socket) => {
         callback();
     });
 
+    socket.on('try_again_event', ({roomId}) => {
```

Comment:
everything looks good! i think you can just remove this since my PR already has the code, it just has to be merged

Reply:
sounds good! will do